src/docs/product/dashboards/widget-builder/index.mdx

```diff
@@ -67,11 +67,11 @@ Choosing "Errors and Transactions" allows you to query and aggregate error and t
 - Throughput by country
 - Users most affected by errors
 
-For performance widgets (based on transactions), [metric-backed](/product/sentry-basics/search/searchable-properties/#metrics-properties) data is the default. If you make a selection in configuring your widget that's not metrics-compatible, then your widget will use indexed events, or event-based data, and a message saying this will be displayed.
+For performance widgets (based on transactions), the data is based on [processed transaction events](/product/sentry-basics/search/searchable-properties/#processed-event-properties) data is the default. If you make a selection in configuring your widget that's not metrics-compatible, then your widget will use indexed events, or event-based data, and a message saying this will be displayed.
```

Comment:
```suggestion
For performance widgets (based on transactions), data based on [processed transaction events](/product/sentry-basics/search/searchable-properties/#processed-event-properties) is the default. If you make a selection in configuring your widget that's not compatible with processed event data, then your widget will use indexed transaction events, or event-based data, and a message saying this will be displayed.
```